scripts/akrtma3d/exakrtma3d_gsianl.ksh

```diff
@@ -91,43 +91,43 @@ RUN_GUST="FALSE"
 #fi
 
 # Link to the prepbufr data
-if [ -r ${OBS_DIR}/rtma.t${cyc}z.prepbufr.tm00 ]; then
-  ${LN} -sf ${OBS_DIR}/rtma.t${cyc}z.prepbufr.tm00 ./prepbufr
+if [ -r ${OBS_DIR}/${NET}.t${cyc}z.prepbufr.tm00 ]; then
+  ${LN} -sf ${OBS_DIR}/${NET}.t${cyc}z.prepbufr.tm00 ./prepbufr
 fi
 
-if [ -r "${OBS_DIR}/rtma3d.t${cyc}z.NSSLRefInGSI.bufr" ]; then
-  ${LN} -sf ${OBS_DIR}/rtma3d.t${cyc}z.NSSLRefInGSI.bufr ./refInGSI
+if [ -r "${OBS_DIR}/${NET}.t${cyc}z.NSSLRefInGSI.bufr" ]; then
```

Comment:
This will need to be ${RUN}, but it was not changed in the obsprep_radar script yet.  Will change in compliancy work.